annotations/forms.py

```diff
@@ -20,6 +20,26 @@ def __init__(self, *args, **kwargs):
 		self.helper = FormHelper(self)
 
 
+class RegistrationForm(forms.Form):
+
```

Comment:
Can you add a docstring here? See my other comment. This can be simpler, just a description of what it is for.

```
"""
This class is for....
"""
```